src/SFA.DAS.Commitments.Api.Client.UnitTests/ApiClientTests/EmployerCommitmentApiTests.cs

```diff
@@ -296,5 +295,16 @@ public void Arrange()
 
             Assert.Pass();
         }
+
+        [Test]
+        public async Task PutApprenticeshipStopDate()
+        {
+            var employerRequest = new TestRequest(new Uri(ExpectedApiBaseUrl + $"api/employer/{EmployerAccountId}/commitments/{CommitmentId}/apprenticeships/{ApprenticeshipId}/stopdate"), JsonConvert.SerializeObject(new ApprenticeshipStopDate()));
+            _fakeHandler.AddFakeResponse(employerRequest, new HttpResponseMessage { StatusCode = HttpStatusCode.OK, Content = new StringContent(string.Empty) });
+
+            await _employerApiClient.PutApprenticeshipStopDate(EmployerAccountId, CommitmentId, ApprenticeshipId, new ApprenticeshipStopDate());
+
+            Assert.Pass();
```

Comment:
As discussed with Paul, this test doesn't add much value (although it follows the pattern of the existing tests). It basically only tests if the code throws an exception, but an exception is only thrown if it's supplied with bad test data, so it's more of a test of the test code rather than the code under test.

It's probably outside the scope of this story, but we could either enhance FakeResponseHandler to allow it to verify that the correct call is made on HttpClient, or even better, remove the hand rolled test code and switch to FakeItEasy (or other mocking framework) that allows you to mock concrete classes. (Moq can't mock HttpClient, because it only supports mocking objects with an appropriate interface.)